packages/report-server/src/auth/middleware.ts

```diff
@@ -1,22 +1,33 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { NextFunction, Response } from 'express';
 import { UnauthenticatedError } from '@tupaia/utils';
 import { AccessPolicy } from '@tupaia/access-policy';
 import { getUserIDFromToken } from '@tupaia/auth';
 import { ReportsRequest } from '../types';
 
 const authenticateUser = async (req: ReportsRequest) => {
-  const authHeader = req.headers.authorization || req.headers.Authorization;
+  const authHeader = req.headers.authorization;
```

Comment:
No need for `req.headers.authorization || req.headers.Authorization;` as `express` maps any auth header to `authorization`, see `@types/http.IncomingHttpHeaders`